src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -206,6 +206,16 @@ static Config::ModelType getModelType(const std::shared_ptr<const Model>& model)
     return Config::ModelType::Unknown;
 }
 
+void Plugin::apply_rt_info(const std::shared_ptr<const ov::Model>& model, ov::intel_cpu::Config& config) const {
+    if (model->has_rt_info({"runtime_options", "KV_CACHE_PRECISION"})) {
```

Comment:
```suggestion
    if (model->has_rt_info({"runtime_options", kv_cache_precision.name()})) {
```
to minimize # of hardcoded strings 

the same for other places.